evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -126,6 +127,23 @@ open class PackageRule(
         }
     }
 
+    fun issue(severity: Severity, message: String) = issue(severity, pkg.id, null, null, message)
```

Comment:
Are these convenience functions intended to be added in this commit?

Reply:
Yes, they are required to keep the API unchanged, because the original functions in `Rule.kt` have changed.